kpatch-build/lookup.c

```diff
@@ -338,7 +341,22 @@ struct lookup_table *lookup_open(char *symtab_path, char *symvers_path,
 
 void lookup_close(struct lookup_table *table)
 {
+	int i;
+	struct object_symbol *obj_sym;
+	struct export_symbol *exp_sym;
+
+	for_each_obj_symbol(i, obj_sym, table) {
+		if (obj_sym->name != NULL)
```

Comment:
The code can be simplified by removing the checks for `NULL`. Calling `free(NULL)` is no-op.

Reply:
yet, I have removed the checks for NULL, please review again, thanks !